src/main/java/liquibase/AwsLicenseService.java

```diff
@@ -100,9 +100,9 @@ private void logErrorOnlyOnce(String message, Exception e) {
     @Override
     public String getLicenseInfo() {
         if (licenseIsValid(LicenseTier.PRO.getSubject())) {
-            return "Liquibase Pro " + BUILD_VERSION + " (licensed through AWS License Manager)";
+            return "Liquibase Secure " + BUILD_VERSION + " (licensed through AWS License Manager)";
         } else {
-            return String.format(LIQUIBASE_OPEN_SOURCE_MSG, BUILD_VERSION);
+            return "";
```

Comment:
Should we show a customized error message here so we can detect possible build problems?